test/common.js

```diff
@@ -77,4 +70,19 @@ module.exports = {
         return response.body;
     },
 
+    async deleteAllViaApi(entities, entityType, { addDefaultVhost = true } = {}) {
+        entities = entities.filter(x => x && x.name && x.name.indexOf('amq.') === -1);
+        console.log(`=============================== CLEANUP: deleting ${entities.length} ${entityType} ===============================`);
+
+        await Promise.each(entities, async entity => {
+            await this.deleteViaApi(entityType, entity.name, { addDefaultVhost });
+        });
+    },
+
+    async deleteViaApi(entityType, name, { addDefaultVhost = true } = {}) {
+        let response = await superagent
```

Comment:
Future-proof for non-default vhosts.
```suggestion
        let vhost = addDefaultHost ? '/' : '';
        let response = await superagent
```